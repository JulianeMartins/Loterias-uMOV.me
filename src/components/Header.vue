<template>
  <header class="pb-3 d-flex justify-content-center align-items-center">
    <img alt="Vue logo" src="../assets/logo.png" />
    <h1 class="m-0">Loterias</h1>
  </header>
</template>
<script>
export default {
  name: "Header",
};
</script>
