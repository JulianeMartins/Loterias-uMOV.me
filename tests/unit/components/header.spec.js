import { mount } from "@vue/test-utils";
import Header from "@/components/Header.vue";

describe("Header", () => {
  test("match snapshot", () => {
    const wrapper = mount(Header);
    expect(wrapper.element).toMatchSnapshot();
  });
});
