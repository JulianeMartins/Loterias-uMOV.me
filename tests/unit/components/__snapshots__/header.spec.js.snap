// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header match snapshot 1`] = `
<header
  class="pb-3 d-flex justify-content-center align-items-center"
>
  <img
    alt="Vue logo"
    src=""
  />
  <h1
    class="m-0"
  >
    Loterias
  </h1>
</header>
`;
